xsd/parse.go

```diff
@@ -807,10 +807,21 @@ func parseAnyElement(ns string, el *xmltree.Element) Element {
 	if typeattr != "" {
 		base = parseType(el.Resolve(typeattr))
 	}
+	namespaceAttr := el.Attr("", "namespace")
+	defaultAttr := el.Attr("", "default")
+	optional := false
```

Comment:
This can assume the zero value for a `bool` - ie `false` without needing to be explicit.
```suggestion
	var optional bool
```

Reply:
updated